keyboards/gmmk/pro/pro.c

```diff
@@ -14,3 +14,177 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 #include "pro.h"
+
+#ifdef RGB_MATRIX_ENABLE
+
+const aw_led g_aw_leds[DRIVER_LED_TOTAL] = {
+/* Refer to AW20216S manual for these locations
```

Comment:
Could you elaborate where to find documentation for that chip?
The official awinic website does not list the AW20216S.